Quadriga/src/main/webapp/resources/js/cytoscape/publicNetwork.js

```diff
@@ -183,11 +189,21 @@ function getTexts(node, path, unixName) {
 				parsedData = JSON.parse(data);
 				var projects = parsedData['projects'];
 				projects.forEach(function(element, index, array) {
-					conceptDesc += "<p class='text-muted' style='margin-bottom:2px'>" + element["text"] + "</p>";
-					conceptDesc += "... " + hightlight(element["textContent"][0], element["phrases"]) + "..." + "<br>";
+					handler += "<a href="+ "'" + element["text"] + "'" + "target='_blank'>" + element["text"] + "</a>";
```

Comment:
this would append the link for every project to the handler variable and then later on put that into the concept description. when there are two projects the second project would have the value from the first project as well as the value from the second project (given that you just append it).